packages/core/src/utilities/font-family/font-family.scss

```diff
@@ -1,38 +1,10 @@
 /* stylelint-disable declaration-no-important */
 @import '@cmsgov/design-system-support/src/settings/index';
 
-/*
-Font family
```

Comment:
I'm wondering if we can combine font family, style, size, and weight into one font page?

Reply:
yeah i was thinking of leaving that for a separate pr. same with border and border radius. margin and padding is another candidate (the examples r pretty much the same). i also think it should be another PR because maybe our vertical nav should be updated to have subsections, or some other change to expose these newly hidden sections

Reply:
yeah i was thinking of leaving that for a separate pr. same with border and border radius. margin and padding is another candidate (the examples r pretty much the same). i also think it should be another PR because maybe we should consider updating our vertical nav to have subsections, or some other change to expose these newly hidden sections